search_service/proxy/elasticsearch.py

```diff
@@ -311,7 +311,7 @@ def fetch_table_search_results_with_field(self, *,
                         "multi_match": {
                             "query": query_term,
                             "fields": ["display_name^1000",
-                                       "name.raw^30",
+                                       "name.raw^75",
```

Comment:
FYI, we don't use this endpoint any more.

Reply:
Sounds good. There's no harm having same weight `75` at this point, right?